src/styles/container.scss

```diff
@@ -0,0 +1,23 @@
+.container {
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  padding: 50px 40px;
+  justify-content: space-between;
+  gap: 46px;
+
+  @media (min-width: 1200px) {
+    padding-inline: 128px;
+  }
```

Comment:
This media query is not necessary as it only changes padding and does not affect the layout structure. It could potentially lead to confusion or inconsistency. Consider removing it.